### DISCUSSION TITLE
Add search feature to Boost GUI storage deals window

### DISCUSSION THREAD

**Discussion by stuberman** [filecoin-project/boost#514] [Ideas]
I would like to see the storage deals be searchable (by client ID, Deal ID or any other feasible means). As a storage provider, we spend a lot of time with the Market node searching the logs with grep for particular deal info.

**Comment by LexLuthr** [comment]
It should be covered in https://github.com/filecoin-project/boost/issues/515

**Reply by stuberman** [reply]
Yeah, that was created because of this discussion.
